Goutte/Client.php

```diff
@@ -129,14 +129,16 @@ protected function doRequest($request)
                 $requestOptions['body'] = $content;
             } else {
                 if ($files = $request->getFiles()) {
-                    $requestOptions['multipart'] = [];
+                    $requestOptions['multipart'] = array();
```

Comment:
What's wrong with the short syntax for arrays?
Goutte requires at least PHP 5.5 and short syntax for arrays are available since PHP 5.4.

Also, this change has nothing to do with adding GET parameters to the request and thats why we should not include that.